### DISCUSSION TITLE
- Final Project - The Women Who Voted For Trump

### DISCUSSION THREAD

**Discussion by giscus** [DACSS/DACSS_601_August2022_v2#262] [Announcements]
# - Final Project - The Women Who Voted For Trump

https://dacss.github.io/DACSS_601_August2022_v2/posts/KarenDetter_FinalProject.html

**Comment by LarriMill** [comment]
Hi Karen, I absolutely love this project. You do a great job clearly describing your questions and process. I would like to see a bit more in depth visualizations, but I recognize that you were limited by the scope of your data. Well done!
